media/CircleEditor/index.js

```diff
@@ -146,6 +149,12 @@ host.BrowserHost = class {
                 case 'reload':
                     this._msgReload(message);
                     break;
+                case 'edit':
+                    this._msgLoadModel(message);
```

Comment:
Only seeing the `command`, `edit` seems not related with loading model.
Furthermore, this case is exactly same with `loadmodel`.
What is this case for?

Reply:
I separated the `edit` command considering the possibility of partial modification in the future, but I think it can be combined into `loadmodel` :)